pmlc/rt/xsmm.cc

```diff
@@ -277,17 +277,87 @@ extern "C" void plaidml_rt_xsmm_binary_invoke(int64_t addr, void *input1,
   kernel(&param);
 }
 
+extern "C" void plaidml_rt_xsmm_brgemm_relu_invoke_f32(int64_t funcAddr,
```

Comment:
@egeor I see this method segfaulting at runtime and am unable to debug it using gdb as it points to some other unrelated error. Can you point out to any obvious mistakes I may have made in supporting ext op with this invoke and the following dispatch methods?